database/migrations/2024_05_28_105201_integrations_previous_statuses.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+declare(strict_types=1);
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class () extends Migration {
+    public function up(): void
+    {
+        Schema::create('integrations_previous_statuses', function (Blueprint $table) {
+            $table->uuid('id')->primary();
```

Comment:
I don't think we need both an `id` and an `integration_id` column. The `id` column can just be used to store the integration ID and that makes it also unique.